sdk/translation/azure-ai-translation-document/tests/test_list_submitted_jobs.py

```diff
@@ -4,77 +4,250 @@
+
+        # create some jobs with the status 'Succeeded'
+        completed_job_ids = self._create_and_submit_sample_translation_jobs(client, jobs_count, wait=True, docs_per_job=docs_per_job)
+
+        # create some jobs with the status 'Cancelled'
+        job_ids = self._create_and_submit_sample_translation_jobs(client, jobs_count, wait=False, docs_per_job=docs_per_job)
+        for id in job_ids:
+            client.cancel_job(id)
+        self.wait(10) # wait for cancelled to propagate
+
+        # list jobs with status filter
+        statuses = ["Cancelled"]
+        submitted_jobs = list(client.list_submitted_jobs(statuses=statuses))
+
+        # check statuses
+        for job in submitted_jobs:
+            self.assertIn(job.status, statuses)
+            self.assertNotIn(job.id, completed_job_ids)
+
+
+    @DocumentTranslationPreparer()
+    @DocumentTranslationClientPreparer()
+    def test_list_submitted_jobs_filter_by_ids(self, client):
+        jobs_count = 3
+        docs_per_job = 2
 
         # create some jobs
-        job_ids = self._create_and_submit_sample_translation_jobs(client, jobs_count)
+        job_ids = self._create_and_submit_sample_translation_jobs(client, jobs_count, wait=False, docs_per_job=docs_per_job)
 
-        # list jobs - unable to assert skip!!
-        submitted_jobs = list(client.list_submitted_jobs(skip=skip))
+        # list jobs
+        submitted_jobs = list(client.list_submitted_jobs(job_ids=job_ids))
+        self.assertIsNotNone(submitted_jobs)
+
+        # check statuses
+        for job in submitted_jobs:
+            self.assertIn(job.id, job_ids)
+
+
+    @DocumentTranslationPreparer()
+    @DocumentTranslationClientPreparer()
+    def test_list_submitted_jobs_filter_by_created_after(self, client):
+        # create some jobs
+        jobs_count = 3
+        docs_per_job = 2
+
+        # create some jobs
+        start = datetime.now()
+        job_ids = self._create_and_submit_sample_translation_jobs(client, jobs_count, wait=False, docs_per_job=docs_per_job)
+
+        # list jobs
+        submitted_jobs = list(client.list_submitted_jobs(created_after=start))
         self.assertIsNotNone(submitted_jobs)
 
+        # check statuses
         for job in submitted_jobs:
-            if job.id in job_ids:
-                self._validate_translation_job(job, status="Succeeded", total=TOTAL_DOC_COUNT_IN_JOB, succeeded=TOTAL_DOC_COUNT_IN_JOB)
-            else:
-                self._validate_translation_job(job)
\ No newline at end of file
+            self.assertIn(job.id, job_ids)
+            assert(job.created_on.replace(tzinfo=None) >= start.replace(tzinfo=None))
+
+
+    @DocumentTranslationPreparer()
+    @DocumentTranslationClientPreparer()
+    def test_list_submitted_jobs_filter_by_created_before(self, client):
+        '''
+            NOTE: test is dependent on 'end' to be specific/same as time zone of the service! 
+                'end' must be timezone-aware!
```

Comment:
Interesting!

Reply:
any time-related filtering actually for that matter!